FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1186,6 +1186,11 @@ components:
           description: Defaults to TIME OF REGISTRATION unless there has been a reset from an administrator to start tracking a different period of time. Tracks the last time that a component was repaired, replaced, or any other maintenance has been performed, thus requiring a reset in the lifetime statistics below / MTBF time. Formatted according to RFC 3339 section 5.6.
           type: string
           format: date-time
+
+        owner:
```

Comment:
Is there a reason this is a UUID and not just the client-id as a string? Where does this UUID come from anyway?

Reply:
This UUID is the ID of the "invisible" user keycloak generates when enabling service accounts. I picked this because should we ever switch to a user based authentication of a client based authentication, this system will still work.